tests/conftest.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pathlib
+import pytest
+import requests
+import types
+
+def download_if_needed(local_path):
```

Comment:
Is it worth adding a check that the user has internet connection to download the data?